rules/windows/execution_ms_office_written_file.toml

```diff
@@ -81,25 +88,28 @@ sequence with maxspan=2h
       process.name : "eqnedt32.exe" or
       process.name : "fltldr.exe" or
       process.name : "MSPUB.EXE" or
-      process.name : "MSACCESS.EXE")
+      process.name : "MSACCESS.EXE") and
+    not file.name : "NewOutlookInstaller.exe"
```

Comment:
if `NewOutlookInstaller.exe` is signed or has few unique `process.hash.sha256` better exclude it in the second event `[process where host.os.type == "windows" and event.type == "start" and not process.hash,sha256 : "1234..."`

Reply:
No hash, but it is signed by Microsoft, so added the additional info there.